packages/rancher-windows-exporter/charts/scripts/run.ps1

```diff
@@ -36,9 +36,43 @@ function Transfer-File
     $null = Copy-Item -Force -Path $Src -Destination $Dst
 }
 
-# Copy binary
-Create-Directory -Path "c:\host\etc\windows-exporter"
-Transfer-File -Src "c:\etc\windows-exporter\windows-exporter.exe" -Dst "c:\host\etc\windows-exporter\windows-exporter.exe"
+# Copy binary into host
+Create-Directory -Path "C:\host\etc\windows-exporter"
+Transfer-File -Src "C:\etc\windows-exporter\windows-exporter.exe" -Dst "C:\host\etc\windows-exporter\windows-exporter.exe"
 
-# Run wins
-Invoke-Expression -Command c:\entry.ps1
+# Copy binary into prefix path, since wins expects the same path on the host and on the container
+$prefixPath = 'C:\'
+if ($env:CATTLE_PREFIX_PATH) {
+    $prefixPath = $env:CATTLE_PREFIX_PATH
+}
+$winsDirPath = $('{0}etc\windows-exporter' -f $prefixPath)
```

Comment:
nit: the path below has a path separator after the variable. Should they both match for UX consistency?

Reply:
No need. The reason why the first doesn’t have a path separator is because the CATTLE_PREFIX_PATH has a \ at the end.

By definition, the winsDirPath does not have a \ at the end so we need to add it to construct winsPath based on it.

Reply:
This is also in a powershell script, so there’s no UX involved